### PR TITLE
Infographic

### PR DESCRIPTION
**Pull request by Creohex**
- Added `ConsumablesAccumulator` (+ `ConsumablesEntry` and `ConsumeStore`) as a single point for per-player calculations (really not sure about naming though), it is reused in `PrintConsumables`, `PrintConsumableTotalsCsv` and newly-added `Infographic`
- Added `Currency` model to make it easier to deal with prices + some tests
- Added `Infographic`
- Refactored readme from plaintext to markdown
- Added .gitignore
- New dependencies, tests, ..

### CODE REVIEW

File: src/melbalabs/summarize_consumes/package.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import importlib.metadata
 
-VERSION = "2024.1062"
+VERSION = "2024.1063"
```
Comment (melbaa) [inline src/melbalabs/summarize_consumes/package.py:3]: the version also gets updated in pyproject.toml.

It's automated via tasks.py. when you type eg
`inv commit 'some commit msg'`.
I usually do it for my last commit before I publish a new version.

File: deps/requirements.txt [hunk 2]
```diff
@@ -4,5 +4,7 @@ humanize==4.8.0           # via -r deps\requirements.in
 idna==3.4                 # via requests
 interegular==0.3.2        # via lark
 lark[interegular]==1.1.8  # via -r deps\requirements.in
+plotly==5.22.0
```
Comment (melbaa) [inline deps/requirements.txt:7]: those 2 new dependencies should also be added unpinned in deps/requirements.in.

File: src/melbalabs/summarize_consumes/main.py [hunk 3]
```diff
@@ -1385,12 +1385,105 @@ def add_extra_attacks(self, howmany, source, name):
...
+
+
+class ConsumablesAccumulator:
+    def __init__(self, player, pricedb, death_count) -> Self:
         self.player = player
         self.pricedb = pricedb
         self.death_count = death_count
+        self.data = {}
+
+    def get_consumable_price(self, consumable):
```
Comment (melbaa) [inline src/melbalabs/summarize_consumes/main.py:1448]: no need to do anything now, just a design comment

making this a member function of this class means some random code now has to instantiate a ConsumablesAccumulator just to get the price of a consumable, which makes 0 sense. 
Another symptom is the function doesn't use the self.data attribute, which is the main purpose of this class.

File: src/melbalabs/summarize_consumes/main.py [hunk 4]
```diff
@@ -1385,12 +1385,105 @@ def add_extra_attacks(self, howmany, source, name):
...
+            price = self.pricedb.lookup(itemid)
+            if not price: continue
+            total_price += int(price * multi)
+
+        total_price /= CONSUMABLE_CHARGES.get(consumable, 1)
+        return total_price
+
+    def calculate(self):
+        self.data = collections.defaultdict(lambda: {
+            'deaths': 0,
```
[the remark this answers is not in the document]
Comment (Creohex) [reply src/melbalabs/summarize_consumes/main.py]: Indeed, it's now considerably more convenient to use those entities. I'm still a little conflicted on naming choice i went with, but that can be changed in no time

**Pull request merged**